### DISCUSSION TITLE
Repeatable group

### DISCUSSION THREAD

**Discussion by warudin** [wpengine/atlas-content-modeler#581] [Ideas]
I've seen that ACM now has the ability to make fields repeatable. This is a real nice addition.

When comparing this feature to Advanced Custom Fields, I miss the ability to group multiple fields and make that whole group repeatable. There's probably several use cases in which being able to clone a complete set of fields is necessary.

Per instance: 
- I create a custom post-type listing 'Interesting Developers'.
- I want to list their accounts on different social media websites (Twitter, Facebook, GitHub, etc.) but I don't know up front which social media websites the user will want to insert
- Per social media-item I want to insert: their handle, actual name, URL, the logo from the social media website and their different fields of expertise.

It would be nice to be able to repeat these different fields as a group.

**Comment by mindctrl** [comment]
Hey @warudin,

Thanks for taking the time to give us feedback and a good use case example. We do have this item in our backlog, but I'm not sure when we'll have something to show.

The way we recommend working with this use case for now is to create another model for social websites, and use the relationships field that connects the "Interesting Developers" with the new social media websites model. We know it's not the best workflow, but it will help you achieve something similar to what you described.

Hope this helps. Let us know if you have questions or thoughts.

**Reply by warudin** [reply]
Hi @mindctrl,

Thanks for your reply. 

I thought about this solution as well. But lets be honest: having another post-type would be far from the most convenient way, neither for the developer or the end-user. 

Per instance, you would have to switch between different browser tabs all the time to create the various social media entries. And over time all social media entries from individual developers will be showing up criss-cross in the back-end which will make administering them a hassle.

So I really hope that this feature request will find some traction with other developers and the development team.

**Reply by nickcernis** [reply]
Thank you again for voting for this feature and sharing your feedback with us, Dominique.

We'll update this thread in the future if we make more concrete plans to add repeatable field groups.
